src/components/organisms/sidebar/sidebar.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable tailwindcss/no-custom-classname */
```

Comment:
When you fix the `size-5.5` you won't need this line

Reply:
fixed